### DISCUSSION TITLE
does degoogling patches in bromite decrease security?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1662] [Q&A]
an hexavalent dev says that degoogling patches in bromite decrease security,is that true???

**Comment by csagan5** [comment]
@TheAnonymous88 you should cite your sources

**Reply by ghost** [reply]
> @TheAnonymous88 you should cite your sources

![Screenshot_20211222-105158](https://user-images.githubusercontent.com/95953932/147039918-3ba652d2-76bb-48ed-ba2f-1d76e965bc94.jpg)

**Reply by chrislogan-foss** [reply]
No it increases your privacy

**Reply by csagan5** [reply]
That is just a quote of someone saying something. You should ask that person to explain why etc

**Comment by Tanzin01** [comment]
Security is a broad term. In internet, there is hardly anything which is secure to use. That's why one get a security update almost every month. Means it is/was never secure.

Adding new and more things break other things, resulting in security and various issues. Atop google itself is unsecure that's why we are here on ungoogled chromi discussion. In short, it's a no if the project is a maintained one.
